spec/defines/download_file_spec.rb

```diff
@@ -229,4 +229,20 @@
       }
     end
   end
+
+  describe 'the proxyAddress parameter' do
+    let(:title)  { 'Download nodejs installer' }
+    let(:params) {{
+        :url => 'http://my.server/test.exe',
+        :destination_directory => 'c:\temp',
+        :destination_file => 'foo.exe',
+        :proxyAddress => 'http://localhost:9090'
+    }}
+
+    describe 'is still supported' do
```

Comment:
I'll leave it for now, we change it after the major version bump to say that a Puppet::Error should happen for the removed parameter.